docs/kane-ai-smart-variables.md

```diff
@@ -0,0 +1,134 @@
+hide_title: false
+sidebar_label: Smart Variables
+description: Learn how to utilize Smart Variables to dynamically adapt to various environments and conditions
+keywords:
+  - lambdatest automation
+  - lambdatest kaneai
+  - kaneai smart variables
+  - kaneai parameters
+url: https://www.lambdatest.com/support/docs/kane-smart-variables/
+site_name: LambdaTest
+slug: kane-ai-smart-variables/
+---
+
+<script type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify({
+       "@context": "https://schema.org",
+        "@type": "BreadcrumbList",
+        "itemListElement": [{
+          "@type": "ListItem",
+          "position": 1,
+          "name": "Home",
+          "item": "https://www.lambdatest.com"
+        },{
+          "@type": "ListItem",
+          "position": 2,
+          "name": "Support",
+          "item": "https://www.lambdatest.com/support/docs/"
+        },{
+          "@type": "ListItem",
+          "position": 3,
+          "name": "KaneAI Smart Variables",
+          "item": "https://www.lambdatest.com/support/docs/kane-ai-smart-variables/"
+        }]
+      })
+    }}
+></script>
+
+KaneAI's Smart Variables feature enhances test automation by providing predefined, dynamic variables that can be used across test steps, modules, and test cases. These variables enable dynamic adaptation to various environments and conditions, improving test reliability, reusability, and efficiency. 
+
+They cover categories such as date & time, system information, location, connectivity, randomization, and device-specific details, allowing users to create flexible and robust test scenarios that adapt to real-time conditions without manual input.
+
+:::note
+ Smart variables are predefined and cannot be created or deleted by users. They are accessible in test steps, modules, and test cases.
+ When typing `{{` in the Web/App Agent, an overlay displays all available variables, including regular, secret, and smart variables, with smart variables labeled as "Smart" and accompanied by descriptions.
+:::
+
+Smart Variables listing is available near User Variables
+
+<img loading="lazy" src={require('../assets/images/kane-ai/features/smart-variables/smart-variables-list.png').default} className="doc_img"/> 
+
+## How to Use Smart Variables in Your Tests?
+
+Smart variables can be referenced in test steps, modules, and test cases using the `{{smart.variable_name}}` syntax. Follow these steps to integrate them into your tests:
+
+### Step 1: Access the Test Environment
+
+Navigate to the Web/App Agent & enter the authoring mode.
+
+### Step 2: Reference a Smart Variable
+
+In the "Write a Step" field type `{{` to trigger the variable overlay. This will display a list of all available variables, including smart variables labeled with "Smart".
+
+For example, to use the current date in a test step:
+
+```
+Enter the value of {{smart.current_date}} in the date field
```

Comment:
Use {{ syntax to get the list of smart variables listed and type to search and filter them out to select the desired one